.codeclimate.yaml

```diff
@@ -1,9 +1,43 @@
-version: "2"
+version: "2"         # required to adjust maintainability checks
+
 checks:
-  similar-code:
-    enabled: false
+  argument-count:
+    enabled: true
+    config:
+      threshold: 4
+  complex-logic:
+    enabled: true
+    config:
+      threshold: 4
   file-lines:
+    enabled: true
     config:
       threshold: 250
-exclude_patterns:
-  # blank for now
+  method-complexity:
+    enabled: true
+    config:
+      threshold: 5
+  method-count:
+    enabled: true
+    config:
+      threshold: 20
+  method-lines:
+    enabled: true
+    config:
+      threshold: 25
```

Comment:
i think for method or function, 100 lines limit is reasonable

Reply:
love it, thank you for the feedback bing